pkg/account/model.go

```diff
@@ -1,22 +1,50 @@
 package account
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/Optum/dce/pkg/arn"
 	"github.com/Optum/dce/pkg/errors"
 	validation "github.com/go-ozzo/ozzo-validation"
 )
 
+var (
+	// PrincipalPolicyName default principal policy Name
+	PrincipalPolicyName string
+)
+
+// ValidStatuses has the valid status options
+var ValidStatuses = [5]Status{
+	StatusNone,
+	StatusLeased,
+	StatusNotReady,
+	StatusOrphaned,
+	StatusReady,
+}
+
+func init() {
+	var ok bool
+	PrincipalPolicyName, ok = os.LookupEnv("PRINCIPAL_POLICY_NAME")
```

Comment:
Why not inject this?

Reply:
That's not easily injectable.
1. As opposed to the Principal Role Name we don't store the policy name in the database.
2. When unmarshaling JSON/DynamoDB I want to populate this value so we can pass it along with the account.  That means I need it outside the struct and I need it some place those functions can get at it.